site/src/layouts/DetailComponent/MobileDrawer.module.css

```diff
@@ -7,7 +7,7 @@
   margin-top: calc(-1 * calc(var(--salt-size-unit) * 2));
   box-shadow: var(--salt-overlayable-shadow-modal);
   padding-left: calc(var(--salt-size-unit) * 2);
-  transition: max-height 1s ease-in-out;
+  transition: max-height 0.6s ease-in-out;
```

Comment:
`--salt-duration-*` variables could help?

Reply:
I was wondering the same thing, but [the documentation](https://storybook.saltdesignsystem.com/?path=/docs/theme-foundations-duration--page) states that they are "duration between the moment a user performs an action and the moment feedback is given by the system".

Since this is the duration of an animation, not the delay before something happens on-screen, I'm not sure they're appropriate.

(Aside: This does make me wonder whether there should be tokens whose intent is how long an animation runs for. Thoughts, @pseys & @dplsek?)

Reply:
(Also: Is it OK for a component to reference _foundation_ tokens? I've always assumed those are kind of private to the theme. Is that not the case?)

Reply:
cc @navkaur76

Reply:
> (Also: Is it OK for a component to reference foundation tokens? I've always assumed those are kind of private to the theme. Is that not the case?)

Yes, foundations are part of the theme and can be used directly, with the exception of color [and the intermediate palette] which are given meaning within characteristics
see tooling/stylelint/correct-theme-token-usage/index.js

Reply:
Ah cool. Thanks for clarifying that @libertymayc

Reply:
I've replaced the animation with `--salt-duration-perceptible`, let me know how that looks :)

Reply:
> I was wondering the same thing, but [the documentation](https://storybook.saltdesignsystem.com/?path=/docs/theme-foundations-duration--page) states that they are "duration between the moment a user performs an action and the moment feedback is given by the system".
> 
> Since this is the duration of an animation, not the delay before something happens on-screen, I'm not sure they're appropriate.
> 
> (Aside: This does make me wonder whether there should be tokens whose intent is how long an animation runs for. Thoughts, @pseys & @dplsek?)

Paul may be able to better assist on this, but generally I think the plan is to treat animation separately from duration - it would instead fall under 'Motion' which is a foundation yet to be created. However, if a duration token is needed as a temporary solution for the drawer, then using duration-perceptible as Jo has applied makes sense. Out of the existing tokens, it feels the most acceptable timing for the drawer to expand/collapse upon interaction.